src/utils/action-handler.js

```diff
@@ -0,0 +1,17 @@
+import _ from "lodash";
+import { Router } from "express";
+import Promise from "bluebird";
+
+import responseMiddleware from "./response-middleware";
+import requireHullMiddleware from "./require-hull-middleware";
+
+export default function actionHandler(handler) {
+  const router = Router();
+  router.use(requireHullMiddleware);
+  router.post("/", (req, res, next) => {
+    return Promise.resolve(handler(req.hull, _.pick(req, ["body", "query"]))).then(next, next);
```

Comment:
I would directly pass the whole `req` object to the handler here. @michaloo do you see any reason not to do so ?